nsxt/nsxt_controller_details.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# coding=utf-8
+#
+# Copyright © 2015 VMware, Inc. All Rights Reserved.
```

Comment:
On all of these files, use the header in place of the copyright and license boiler plate down to line 19:

```suggestion
# Copyright © 2015 VMware, Inc. All Rights Reserved.
# SPDX-License-Identifier: Apache 2.0
```